ape_infura/provider.py

```diff
@@ -92,6 +105,18 @@ def connect(self):
     def disconnect(self):
         self._web3 = None
 
+    def reconnect(self):
+        """
+        Disconnect the connected API.
+        Refresh the API keys from environment variable.
+        Make the self.network_uris empty otherwise the old network_uri will be returned.
+        Connect again.
+        """
+        self.disconnect()
+        self.load_api_keys()
+        self.network_uris = {}
+        # self.connect()
```

Comment:
Commented out code here! Let's delete

```suggestion
        # self.connect()
```